Tasks/app-store-release/task.json

```diff
@@ -312,6 +312,7 @@
     },
```

Comment:
Bump the task version.

Reply:
It has been bumped to 147 in previous PR, so skipping in this. Next release of the extension will have *147.0 version of tasks.